### ISSUE TITLE
Standard_name_mapping not working

### ISSUE THREAD

**Issue by ocecept** [OpenDrift/opendrift#404]
Hello;

I am trying to see particles at the bottom of my domaing using z='seaflor', however I am using a ROMS forcing and I do not have sea_floor_depth_below_sea_level in my input file. So, following the Macondo exemplo, I tried to use my batimetry variable (h) using:

my = reader_netCDF_CF_generic.Reader('my_his.nc', standard_name_mapping={'sea_floor_depth_below_sea_level': 'h'})

And I got the following error: TypeError: __init__() got an unexpected keyword argument 'standard_name_mapping'

I also downloaded the exemplo_macondo.py and tried to run it, but I got the same error: 

INFO: OpenDriftSimulation initialised (version 1.3.2)
Traceback (most recent call last):
  File "example_macondo.py", line 18, in <module>
    reader_globcurrent = reader_netCDF_CF_generic.Reader(
TypeError: __init__() got an unexpected keyword argument 'standard_name_mapping'

I could not trace the problem! 

By the way, I know I can create the  sea_floor_depth_below_sea_level using NCO, but I am not sure if it is right to use sea_floor_depth_below_sea_level=bathymetry. Any clues? 

Many thanks and thank you for the great working with OpenDrift!

Carlos Teixeira

**Comment by knutfrode** [comment]
Hi,

This looks a bit strange, since you are using a quite recent version (1.3.2).
Maybe there are some modifications in your local copy of reader_netCDF_CF_generic?

You can check this with `git status` or `git diff` from within your opendrift-folder.
You can eventually restore this file to the official version with `git checkout reader_netCDF_CF_generic` from within the readers-folder.

**Comment by ocecept** [comment]
Hi, 
I checked the reader_netCDF_CF_generic and it was not changed. I also did: git checkout reader_netCDF_CF_generic from within the readers-folder as suggested.  I got the same error. 

But I just saw the new opendrift version 1.3.3. Downloaded it and now example_maconda works fine.. strange.. 

I will try my application setting standard_name_mapping={'sea_floor_depth_below_sea_level': 'h'} and see if it works. 

Thanks a lot!

Carlos
